src/metakb/schemas/app.py

```diff
@@ -1,9 +1,32 @@
 """Module containing app schemas and enums"""
-from enum import Enum
+from enum import StrEnum
 
 
-class SourceName(str, Enum):
+class SourceName(StrEnum):
     """Define enum for sources that are supported"""
 
     CIVIC = "civic"
     MOA = "moa"
+
+    def as_print_case(self) -> str:
+        """Provide enum value with natural print casing (i.e. value-specific for
+        natural text).
+
+        :return: correctly-cased string
+        :raise ValueError: if called on a source value that we forgot to define here
+        """
+        if self == "civic":
+            return "CIViC"
+        if self == "moa":
+            return "MOA"
+        raise ValueError
+
+    def __repr__(self) -> str:
+        """Print as simple string rather than enum wrapper, e.g. 'civic' instead of
+        <NormalizerName.CIVIC: 'civic'>.
+
+        Makes Click error messages prettier.
+
+        :return: formatted enum value
+        """
+        return f"'{self.value}'"
```

Comment:
Bigger explanation of this:

For the most part, when you give enum values to `click.Choice`, it'll format them correctly. For example:

```
[ cli-refactor ⚙ .venv] ~/code/metakb % metakb update --help
Usage: metakb update [OPTIONS] [[civic|moa]]...
```

However, if you pass an invalid arg, it prints them using `__repr__`, so it's ugly:

```
Error: Invalid value for '[[civic|moa]]...': 'sdflkd' is not one of <NormalizerName.CIVIC: 'civic'>, <NormalizerName.MOA: 'moa'>.
```

I don't think there are any major consequences to overwriting `__repr__` but when debugging you have to be a little careful because it'll print without the enum wrapping.